### PR TITLE
Reset password instructions

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #535

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@neerajdotname _a Please check.

**Comment by neerajdotname** [comment]
@prathamesh-sonpatki _a

File: config/initializers/session_store.rb [hunk 1]
```diff
@@ -1 +0,0 @@
-Rails.application.config.session_store :cookie_store, :key => '_acehelp_session', expire_after: 30.minutes
```
Comment (prathamesh-sonpatki) [inline config/initializers/session_store.rb]: Why is this removed?

**Comment by prathamesh-sonpatki** [comment]
- [ ] Setup email using mailgun same as EII
- [ ] Add error handling for non-existent email, wrong email addresses.

@tejaswinichile _A

File: config/initializers/session_store.rb (hunk 1 above)
Comment (prathamesh-sonpatki) [reply config/initializers/session_store.rb]: This is fine as warden is taking care of session.

**Comment by tejaswinichile** [comment]
No need for this change, We are handling it with devise.
Closing this PR.
CC @prathamesh-sonpatki .
